library/elliptics.h

```diff
@@ -1035,8 +1035,7 @@ static inline int dnet_read_ll(int fd, char *data, size_t size, off_t offset)
 /*
  * Watermarks for number of bytes written into the wire
  */
-#define DNET_SERVER_SEND_WATERMARK_HIGH		(30*1024*1024*1024L)
-#define DNET_SERVER_SEND_WATERMARK_LOW		DNET_SERVER_SEND_WATERMARK_HIGH / 2
+#define DNET_SERVER_SEND_WATERMARK_HIGH		(100*1024*1024L)
```

Comment:
JFYI, new server-send recovery speed is bounded by network bandwidth (120 mb/sec) for 128kb+-10kb keys and those numbers are taken from current performance test results of new s.s. recovery.